homeassistant/helpers/entity.py

```diff
@@ -327,6 +333,19 @@ def async_write_ha_state(self) -> None:
 
         self._async_write_ha_state()
 
+    def _stringify_state(self) -> str:
```

Comment:
Why extract it to a method?

Reply:
To clean up the code (add guard conditions to return early)